app/views/handlers/question.py

```diff
@@ -73,7 +75,9 @@ def rendered_block(self):
 
         self._set_page_title(page_title)
         rendered_question = self.placeholder_renderer.render(
-            transformed_block["question"], self._current_location.list_item_id
+            transformed_block["question"],
```

Comment:
Same as my comment above, do what you did in list collector block, keyword attributes?

Reply:
Updated 👍